Dockerfile

```diff
@@ -29,4 +29,4 @@ COPY public ../public
 COPY src ../src
 ENV PORT=8080
```

Comment:
I think this works as Cloud Run defaults to 8080, but you should probably use the environmental variable passed in by Cloud Run and not hardcode it here: https://cloud.google.com/run/docs/reference/container-contract#port